### PR TITLE
Delete fontawesome and repalce it with base64 icons

### PR DESCRIPTION
**Pull request by polyakusha**
screenshot 2019-02-06 at 12 20 03

### CODE REVIEW

File: ui/src/app.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {HttpLink} from "apollo-boost";
 
 
 const client = new ApolloClient({
- link: new HttpLink({ uri: "/query" }),
+ link: new HttpLink({ uri: "http://localhost:8080/query" }),
```
Comment (jeff-knurek) [inline ui/src/app.jsx]: this shouldn't be merged in with this 

- ideally it can be fixed properly with webpack

Comment (polyakusha) [reply ui/src/app.jsx]: Sorry, it shouldn't be in this PR :)

Comment (polyakusha) [reply ui/src/app.jsx]: Fixed!

File: ui/src/app.jsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ import {HttpLink} from "apollo-boost";
 
 
 const client = new ApolloClient({
- link: new HttpLink({ uri: "/query" }),
+ link: new HttpLink({ uri: "query" }),
```
Comment (jeff-knurek) [inline ui/src/app.jsx]: not sure if the `/` should be removed either...

**Pull request merged**